fivestar.js

```diff
@@ -34,9 +34,15 @@ function fivestar_compute_rating(star, base) {
  */
 function fivestar_container_id(entity_type, entity_id, delta) {
   try {
+
+    //
+    // Allow to have more than one fivestar field on node_vew
+    //
+
+    var x = Math.floor((Math.random() * 999999) + 1);
     var id = 'new';
     if (entity_id) { id = entity_id; }
-    return 'fivestar_' + entity_type + '_' + entity_id + '_' + delta;
+    return 'fivestar_' + entity_type + '_' + entity_id + '_' + delta + x;
```

Comment:
@devasghar Thank you for this pull request. I'd like to see the `field_name` used here instead of a random number, then the `id` will be much more predictable for jQuery/CSS selectors. The field name will be available while iterating over the `elements` in `fivestar_form_alter()`. Would you please adjust this pull request for that? Thank you.